tests/e2e/framework/testInfo.go

```diff
@@ -51,7 +51,7 @@ var (
 const (
 	tmpPrefix   = "istio.e2e."
 	idMaxLength = 36
-	pageSize    = 100 // number of log entries for each paginated request to fetch logs
+	pageSize    = 500 // number of log entries for each paginated request to fetch logs
```

Comment:
As we discussed earlier, the issue here is that the log api endpoint rejects our request if requests are sent too frequently. Yutong and I agree that we can try a really large number.